src/main/java/com/northconcepts/datapipeline/examples/parquet/WriteAParquetFileUsingSchemaFromData.java

```diff
@@ -0,0 +1,42 @@
+package com.northconcepts.datapipeline.examples.parquet;
+
+import java.io.File;
+
+import com.northconcepts.datapipeline.core.DataReader;
+import com.northconcepts.datapipeline.core.DebugReader;
+import com.northconcepts.datapipeline.core.StreamWriter;
+import com.northconcepts.datapipeline.csv.CSVReader;
+import com.northconcepts.datapipeline.job.Job;
+import com.northconcepts.datapipeline.parquet.ParquetDataReader;
+import com.northconcepts.datapipeline.parquet.ParquetDataWriter;
+
+public class WriteAParquetFileUsingSchemaFromData {
+
+    private static final File PARQUET_FILE = new File("example/data/output/WriteAParquetFileUsingSchemaFromData.parquet");
+
+    public static void main(String[] args) {
+        System.out.println("============================================================");
+        System.out.println("Write records to a parquet file");
+        System.out.println("============================================================");
+
+        DataReader reader = new CSVReader(new File("example/data/input/bank_account.csv"))
+                .setFieldNamesInFirstRow(true);
```

Comment:
Let's use BasicTransformer to convert values to different data type. Currently all values are string data type. Add a comment for the same.